src/main/java/com/my/relink/util/api/ApiResult.java

```diff
@@ -1,7 +1,27 @@
 package com.my.relink.util.api;
 
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.my.relink.ex.ErrorCode;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @Getter
-public class ApiResult {
+@AllArgsConstructor
+@JsonPropertyOrder({"success", "data", "error"})
+public class ApiResult<T> {
+    private final T data;
+    private final boolean success;
+    private final ApiError error;
+
+    public static <T>ApiResult<T> success(T data){
```

Comment:
```suggestion
    public static <T> ApiResult<T> success(T data) {
```
nit; 중요하진 않지만 띄어쓰기는 통일시키는게 좋아보입니다 ㅎㅎ